installing/installing_ibm_power/creating-ibm-power-vs-service-instance.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: PROCEDURE
+[id="creating-ibm-power-vs-service-instance"]
+= Creating an IBM Power VS service instance
+
+.Procedure
+
+. For detailed steps on creating a Power Systems Virtual Server, see the IBM Cloud documentation for link:https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-creating-power-virtual-server[Creating a Power Systems Virtual Server].
```

Comment:
Can we modify the text here to "To create a Power Systems Virtual Server Service Instance, complete steps 1-5 fro the following documentation:"?

Two things to be careful about here: 
1) that doc is telling users how to create a _server_ (aka VM), which we don't want them to do. Stop after the service instance is created, iow. 
2) Be cognizant of when you use "Virtual Server" vs "Service Instance" (holds VMs, networks, boot images, etc. in a specific zone)